### DISCUSSION TITLE
Can Authelia work without internet access?

### DISCUSSION THREAD

**Discussion by RefineryX** [authelia/authelia#5579] [Q&A] [answered]
I have blocked internet internet to most docker containers including Authelia as I just use these internally. I however keep getting a repeating log item below:

> time="2023-06-17T09:16:38+01:00" level=error msg="Request timeout occurred while handling request from client." error="read tcp 172.28.0.3:9091->172.28.0.2:39136: i/o timeout" method=GET path=/ remote_ip=172.28.0.2 stack="github.com/authelia/authelia/v4/internal/server/handlers.go:71 handleError.func2\ngithub.com/valyala/fasthttp@v1.43.0/server.go:2824             (*Server).writeErrorResponse\ngithub.com/valyala/fasthttp@v1.43.0/server.go:2266             (*Server).serveConn\ngithub.com/valyala/fasthttp@v1.43.0/workerpool.go:224          (*workerPool).workerFunc\ngithub.com/valyala/fasthttp@v1.43.0/workerpool.go:196          (*workerPool).getCh.func1\nruntime/asm_amd64.s:1594                                       goexit" status_code=408

**Comment by james-d-elliott** [comment, marked as answer]
Yes, that error just  indicates a packet from IP 172.28.0.2 took longer than the configured sever timeout to send all of the request data. Usually it can be ignored or you can [increase the timeouts](https://www.authelia.com/configuration/miscellaneous/server/#timeouts).

**Reply by RefineryX** [reply]
Thanks @james-d-elliott. 

From the error above, it appears that it is the 'read' that is timing out. Knowing that the [default](https://www.authelia.com/configuration/prologue/common/#server-timeouts) is 6s, I increased this to 15s and still getting the timeouts.

Should I further increase this or do you reckon anything else is going on? Can I do anything to prevent the timeout?
